cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/SparkTwillRunnable.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.internal.app.runtime.distributed;
+
+import co.cask.cdap.internal.app.runtime.spark.SparkProgramRunner;
+import org.apache.twill.api.TwillRunnable;
+import org.spark_project.protobuf.GeneratedMessage;
+
+/**
+ * A {@link TwillRunnable} wrapper for {@link SparkProgramRunner}.
+ */
+public class SparkTwillRunnable extends AbstractProgramTwillRunnable<SparkProgramRunner> {
+
+  // Dummy vars for dependency tracing
```

Comment:
not sure I understand. Has this to do with making sure that Twill includes GeneratedMessage in the jar? In that case, why not say it in the comment... or if it means something else?